app/home/homeController.js

```diff
@@ -8,7 +8,9 @@
 
     function controllerFunction (cityService) {
           var vm = this;
-          vm.city = cityService.city;
+          vm.cityService = cityService;
+          // Curious as to why you need the whole object. Commented out in forecastController.
```

Comment:
(It's mostly a best practice to expose only the parts of a model, or service, that the view needs.)